### PR TITLE
Web 746

### PR DESCRIPTION
**Pull request by taylorgp**

### CODE REVIEW

File: app/(company)/run-for-office/components/Features.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            <div className="flex flex-col flex-auto items-center text-center mt-10 md:mt-0">
+              <h2 className="font-black text-8xl">50+</h2>
+              <div className="text-lg mt-3">Yeas of Experience</div>
+            </div>
+
+            <div className="flex flex-col flex-auto items-center text-center mt-10 md:mt-0">
+              <h2 className="font-black text-8xl">$0</h2>
+              <div className="text-lg mt-3">Cost to you</div>
+            </div>
+          </div>
```
Comment (tomer-tgp) [inline app/(company)/run-for-office/components/Features.js:49]: you should use a grid here

File: app/(company)/run-for-office/components/Features.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        </div>
+
+        <div className="flex flex-col md:flex-row justify-center items-center mt-10 md:mt-20">
+          <div className="flex flex-col justify-center flex-auto md:w-1/3">
```
Comment (tomer-tgp) [inline app/(company)/run-for-office/components/Features.js:53]: use a grid here too

File: app/(company)/run-for-office/components/Features.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                      </div>
+                    ) : features[index].name === 'Track' ? (
+                      <div className="flex items-center justify-center rounded-full p-2 bg-gradient-to-br from-[#FF3AEB] to-[#3AF3FF] opacity-50">
+                        <RiLineChartLine />
+                      </div>
+                    ) : (
+                      <></>
+                    )
+                  }
+                  children={features[index].description}
```
Comment (tomer-tgp) [inline app/(company)/run-for-office/components/Features.js:75]: why not add the icon and bg to the array and just iterate over it?

File: app/(company)/run-for-office/components/Features.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+          </div>
+        </div>
+        <div className="flex items-center flex-col w-full">
+          <h3 className=" font-black text-4xl mt-10 mb-12">
+            Any questions? Schedule a demo with our team
+          </h3>
+          <div>
+            <div onClick={demoCallback} id="experts-demo-btn">
+              <Pill outlined className="w-48">
+                <div className="tracking-wide">GET A DEMO</div>
```
Comment (tomer-tgp) [inline app/(company)/run-for-office/components/Features.js:96]: grid

File: app/(company)/run-for-office/components/HeroImages.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+// import heroLargeImg from 'public/images/run-page/hero-large.svg';
+
+export default function HeroImages() {
+  return (
+    <div className="flex flex-auto h-[600px] mt-20 md:-mt-20 justify-center relative">
+      <Image
+        src="/images/run-page/hero-large.png"
+        alt="Good Party"
+        fill
+        sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
```
Comment (tomer-tgp) [inline app/(company)/run-for-office/components/HeroImages.js:11]: nice!

**Review by tomer-tgp — CHANGES_REQUESTED** [review]

**Review by tomer-tgp — APPROVED** [review]

File: app/(company)/run-for-office/components/Blog.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+import MaxWidth from '@shared/layouts/MaxWidth';
+import Pill from '@shared/buttons/Pill';
+import Image from 'next/image';
+
+export default function Blog({ articles }) {
+  return (
+    <MaxWidth>
+      <div className="flex items-center justify-center align-middle flex-col w-full mb-12">
+        <h3 className=" font-black text-4xl mt-10 mb-12">Our Blog</h3>
+        <div className="grid grid-cols-1 lg:grid-cols-3 gap-6">
```
Comment (tomer-tgp) [inline app/(company)/run-for-office/components/Blog.js:11]: please keep the grid the same way I use it everywhere:
use `grid grid-cols-12` at the container
and in the children use
`col-span-12 lg:col-span-4`

File: app/(company)/run-for-office/components/Hero.js [hunk 7]
```diff
@@ -1,39 +1,56 @@
...
+            src="/images/run-for-office/hero-star.svg"
+            width="120"
+            height="120"
+            className="mt-5"
+          />
+        </div>
+      </div>
+
+      <div className="grid grid-cols-12 md:gap-10 pl-8 pr-8">
+        <div className="col-span-12 md:col-span-6">
```
Comment (tomer-tgp) [inline app/(company)/run-for-office/components/Hero.js:30]: This is the way
https://i.kym-cdn.com/photos/images/newsfeed/001/653/546/4b2.jpg

**Review by tomer-tgp — CHANGES_REQUESTED** [review]
one more small change

**Comment by taylorgp** [comment]
please merge whenever you're ready.

**Review by tomer-tgp — APPROVED** [review]

**Pull request merged**